util/encoding/encoding_test.go

```diff
@@ -643,6 +646,60 @@ func TestEncodeDecodeNull(t *testing.T) {
 	}
 }
 
+func TestEncodeDecodeTime(t *testing.T) {
```

Comment:
This is a fairly complex and highly-abstracted test. I'd like to see at least one test that is very simple: encode a static time value and compare its encoded form to a static byte string.